Lib/unittest/test/testmock/testasync.py

```diff
@@ -3,7 +3,7 @@
 import unittest
 
 from unittest.mock import (ANY, call, AsyncMock, patch, MagicMock,
-                           create_autospec, _AwaitEvent)
+                            create_autospec, _AwaitEvent, sentinel)
```

Comment:
why the space?

Reply:
Oh, I did a merge in the UI, didn't notice the space